auto_nag/bzcleaner.py

```diff
@@ -382,6 +382,9 @@ def get_comments(self, bugs):
             Bugzilla(
                 bugids=bugids, commenthandler=self._commenthandler, commentdata=bugs
             ).get_data().wait()
+            for bug in bugs.values():
+                if "last_comment" not in bug:
+                    bug["last_comment"] = ""
```

Comment:
When can this happen in practice?

Reply:
No idea, it's a workaround for the issue you met.

Reply:
Let's remove it then, hopefully it was something wrong on my machine. We can add it back in case it fails on the VM too.